src/lib/input/input-container.ts

```diff
@@ -356,10 +357,7 @@ export class MdInputContainer implements AfterViewInit, AfterContentInit {
     @Optional() private _parentFormGroup: FormGroupDirective) { }
 
   ngAfterContentInit() {
-    if (!this._mdInputChild) {
-      throw new MdInputContainerMissingMdInputError();
-    }
-
+    this._validateInputChild();
```

Comment:
not really related to this PR, but I see `_prefixChildren` and `suffixChildren` are unused, want to just get rid of them?